caluma/workflow/models.py

```diff
@@ -104,7 +106,18 @@ class WorkItem(UUIDModel):
         Task, on_delete=models.DO_NOTHING, related_name="work_items"
     )
     status = models.CharField(choices=STATUS_CHOICE_TUPLE, max_length=50, db_index=True)
-    meta = JSONField(default={})
+    meta = JSONField(default=dict)
+
+    addressed_groups = ArrayField(models.CharField(max_length=150), default=list)
+    """
+    Offer work item to be processed by a group of users, such are not committed to process
```

Comment:
Wouldn't it make sense to use the `help` field parameter instead for this?

Reply:
Also, what does "committed" mean here? If I give an item to a group of people, the word "required" would probably be a better match.

Reply:
Good point with help attribute. I will change this.

required would mean someone has to take care of such work items. But this is not the case it is simply saying whether a user of a group wants to take care. If they want to they can ignore this offer. Hence used address.

Reply:
move comment to `help_text`. Now it is even visible in GraphQL documentation schema :+1: